Resources/Locale/en-US/components/hands-component.ftl

```diff
@@ -0,0 +1,8 @@
+### HandComponent stuff
+
+#Examine text after when they're holding something (in-hand)
+comp-hands-examine = {GENDER($user) ->
+    [male] He is
+    [female] She is 
+    *[other] They are
+} holding a {$item}.    
```

Comment:
```suggestion
comp-hands-examine = { CAPITALIZE(SUBJECT($user)) } { CONJUGATE-BE($user) } holding a { $item }.    
```

need some kind of fluent function for "a" vs "an" but thats out of scope for this PR